package.json

```diff
@@ -47,30 +47,31 @@
     "build"
   ],
```

Comment:
Thanks for making these upgrades. I'm aware it's tedious but I recommend walking each of these changelogs to understand what the impact will be on our code and adding it to the commit message to assure reviewers. (My apologies if you've already done this and I missed it!) Otherwise, we're kind of crossing our fingers hoping that tests and manual testing catch any issues, and are unaware of any new features that we could be making use of either as part of the upgrade or as a future change. Here are some example commit messages:

https://gerrit.wikimedia.org/r/#/c/378645/
https://gerrit.wikimedia.org/r/#/c/384111/